src/Symfony/Component/AssetMapper/ImportMap/Resolver/JspmResolver.php

```diff
@@ -96,4 +96,13 @@ public function resolvePackages(array $packagesToRequire): array
             throw $e;
         }
     }
+
+    private function extractVersion(string $url): ?string
+    {
+        if (1 === preg_match("#^https://ga.jspm.io/npm:(?<package>(?:@[a-z0-9-~][a-z0-9-._~]*/)?[a-z0-9-~][a-z0-9-._~]*)@(?<version>[\w\._-]+)(?<subpath>/.*)?$#", $url, $matches)) {
```

Comment:
Not sure that this regex is enough to handle all possible resolved URLs...

Reply:
That's ok - if you have a few test cases to cover the cases we know about, it should be good enough. If we find another case later, we can fix it.

Reply:
OK. Tests are ready to accept new URLs in their data providers.